CardanoSharp.Wallet/Extensions/Models/Transactions/TransactionOutputExtensions.cs

```diff
@@ -137,6 +137,51 @@ public static byte[] Serialize(this TransactionOutput transactionOutput)
             return transactionOutput.GetCBOR().EncodeToBytes();
         }
 
+        public static List<Asset> AggregateAssets(this List<TransactionOutput> transactionOutputs)
+        {
+            List<Asset> assets = new List<Asset>();
```

Comment:
Speaking from some experience, having a container type with both the lovelace value and a collection of Assets makes calculations easier - e.g. no checking for nulls and extra loops to get the lovelace value

```c#
public class Value
{
    public ulong LovelaceValue {get;set;}
    public List<Asset> Assets {get;set;}
}
```

Reply:
Honestly i'd have to review the impact of this type of change. I think this will add more conditions to the implementation since we would be treating assets and lovelaces differently